benchmarks/bun/bench.js

```diff
@@ -12,14 +12,15 @@ import { dirname } from 'path';
 import { fileURLToPath } from 'url';
 import systeminfo from "../systeminfo.js";
 const __dirname = dirname(fileURLToPath(import.meta.url));
-const resultsPath = '/var/results/bun';
+const resultsPath = '../results/bun';
```

Comment:
This is to allow local testing and not to rely on `/var` across platforms.

Reply:
could have been just a ln on local dev.. anyway the docker container itsself could just mount into /var/results.. its ok for me - but still need a change because of the working dir.

Reply:
Oh true. I can make it relative to import.mega.url